packages/pharos/src/components/checkbox/pharos-checkbox.ts

```diff
@@ -55,12 +55,14 @@ export class PharosCheckbox extends FormMixin(FormElement) {
   @query('#checkbox-element')
   private _checkbox!: HTMLInputElement;
 
+  private _defaultChecked = false;
```

Comment:
I was having a hard time getting the `_defaultChecked` to work when using the boolean attributes for both the checkbox and switch components. It seemed the reset wasn't working properly or reset was resetting too much based off the output when running the failing unit tests. 

I saw that `defaultChecked` was an attribute of the [HTMLInputElement](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement/defaultChecked) but by extracting the state to the component, I had much better behavior in the unit tests.